packages/plugin-image/src/renderer.tsx

```diff
@@ -6,7 +6,7 @@ import { imageState } from '.'
 import { ImageLoaded } from './upload'
 
 const Img = styled.img({
-  maxWidth: '100%',
+  width: '100%',
```

Comment:
Why change this from `maxWidth` to `width`? This will probalby look strange for small pictures.